src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
SCSS files should not be linked directly in HTML. You need to compile SCSS to CSS and then link the compiled CSS file.